docs/rst/tutorials/cloud/router_participant_security_via_xml_profiles/router_participant_security_via_xml_profiles.rst

```diff
@@ -0,0 +1,215 @@
+Participants security can only be configured by XML, following the instructions of `Fast DDS Security Documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/security.html>`_ to fulfill the pertinent requirements of the chosen security mechanisms.
+To generate the security certificates needed for both XML Participants follow this `ROS 2 Tutorial <https://docs.ros.org/en/iron/Tutorials/Advanced/Security/Introducing-ros2-security.html>`_.
+
+Once the certificates are generated, introduce their paths in the XML Participants configuration:
+
+.. tabs::
+
+    .. tab:: Participant A
+
+        .. literalinclude:: /resources/tutorials/cloud/router_participant_security_via_xml_profiles/configurations/secure_configurationA.xml
+            :language: xml
+
+    .. tab:: Participant B
+
+        .. literalinclude:: /resources/tutorials/cloud/router_participant_security_via_xml_profiles/configurations/secure_configurationB.xml
+            :language: xml
+
+These XMLs snippet configure security settings for two participants in the Fast DDS middleware, focusing on authentication through the `DDS:Auth:PKI-DH plugin <https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/auth_plugin/auth_plugin.html>`_.
+They activate the PKI-DH plugin, specifying the paths to the participants' identities certificates, identity CA certificate, and private keys for authentication.
+Additionally, they enable access control through the `DDS:Access:Permissions plugin <https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/access_control_plugin/access_control_plugin.html>`_, defining paths to the permissions CA certificate, governance files, and permissions files.
+The configuration also includes the activation of the `DDS:Crypto:AES-GCM-GMAC <https://fast-dds.docs.eprosima.com/en/latest/fastdds/security/crypto_plugin/crypto_plugin.html>`_ plugin for data encryption, enhancing communication security within the Fast DDS framework.
+
+Running ROS 2 Router
+^^^^^^^^^^^^^^^^^^^^
+
+Once all the files needed have been generated, create a workspace called ``workspace_ddsrouter-tutorial`` to collect all the directories and files.
+The tutorial workspace will have the following structure at the end of the project:
+
+.. code-block:: bash
+
+    workspace_ddsrouter-tutorial
+    ├── configurations
+    │   ├── secure_configurationA.xml
+    │   ├── secure_configurationB.xml
+    ├── keystore
+    |   ├── enclaves
+    |   ├── private
+    |   ├── public
+    └── ddsrouter.yaml
+
+The certificates generated are set up to ``ROS_DOMAIN_ID=0`` by default.
+We can set up another domain or even a range of domains applying the following change in ``<domain></domain>`` in the *governance.xml* and in the *permissions.xml* file both for the *talker* and the *listener*:
```

Comment:
I would warn the user that in the permissions.xml files there are two part of the code that need to be changed